src/index.ts

```diff
@@ -26,38 +27,73 @@ const run = async (): Promise<void> => {
+      name: 'Lines',
+      coverage: `${prData.coverage.lines.percent.toFixed(2)}%`,
+      differenceAfterPR: prData.message
+        ? `${prData.message.split('\n')[4].split('|')[3].replace(/ /g, '')} ${prData.message
+            .split('\n')[4]
+            .split('|')[4]
+            .replace(/ /g, '')} `
+        : 'No Data',
+    };
 
-    if (prData.coverage.functions.diff || prData.coverage.functions.diff === 0) {
-      console.log(`Functions difference: ${prData.coverage.functions.diff}`);
-    }
+    const functionsCoverage = {
+      name: 'Functions',
+      coverage: `${prData.coverage.functions.percent.toFixed(2)}%`,
+      differenceAfterPR: prData.message
+        ? `${prData.message.split('\n')[5].split('|')[3].replace(/ /g, '')} ${prData.message
+            .split('\n')[5]
+            .split('|')[4]
+            .replace(/ /g, '')} `
+        : 'No Data',
+    };
 
-    if (prData.coverage.branches.diff || prData.coverage.branches.diff === 0) {
-      console.log(`Branches Difference: ${prData.coverage.branches.diff}`);
-    }
+    const branchesCoverage = {
+      name: 'Branches',
+      coverage: `${prData.coverage.branches.percent.toFixed(2)}%`,
+      differenceAfterPR: prData.message
+        ? `${prData.message.split('\n')[6].split('|')[3].replace(/ /g, '')} ${prData.message
+            .split('\n')[6]
+            .split('|')[4]
+            .replace(/ /g, '')} `
+        : 'No Data',
+    };
 
-    if (prData.message) {
-      console.log(`Message: ${prData.message}`);
+    if (actionDebug === 'true') {
+      console.log(`Repo ID: ${prData.repositoryId}`);
+      console.log(`Ref of branch being merged: ${prData.ref}`);
+      console.log(`Ref of branch being merged into: ${prData.baseRef}`);
+      console.log(`SHA of merge commit: ${prData.sha}`);
+      console.log(`PR creator: ${prData.actor}`);
+      console.log(`Time PR created: ${prData.timestamp}`);
+      console.log(`Lines percent: ${prData.coverage.lines.percent}`);
+      console.log(`Functions percent: ${prData.coverage.functions.percent}`);
+      console.log(`Branches percent: ${prData.coverage.branches.percent}`);
+
+      if (prData.coverage.lines.diff || prData.coverage.lines.diff === 0) {
+        console.log(`Lines difference: ${prData.coverage.lines.diff}`);
+      }
+
+      if (prData.coverage.functions.diff || prData.coverage.functions.diff === 0) {
+        console.log(`Functions difference: ${prData.coverage.functions.diff}`);
+      }
+
+      if (prData.coverage.branches.diff || prData.coverage.branches.diff === 0) {
+        console.log(`Branches Difference: ${prData.coverage.branches.diff}`);
+      }
+
+      if (prData.pullRequest) {
+        console.log(`Pull Request Number: ${prData.pullRequest}`);
+      }
     }
 
-    if (prData.pullRequest) {
-      console.log(`Pull Request Number: ${prData.pullRequest}`);
+    if (prData.message) {
+      console.table([linesCoverage, functionsCoverage, branchesCoverage]);
```

Comment:
I think we should extract `linesCoverage`, `functionsCoverage` and `branchesCoverage` into a function that returns the three of these as a list. IMO it would be easier to follow and less clutter in the main `index` file.
Not a big deal though, just a suggestion